pkg/cmd/cli/cmd/process.go

```diff
@@ -178,8 +178,7 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 			if len(templateName) > 0 {
 				sourceName = namespace + "/" + templateName
 			}
-			fmt.Fprintf(cmd.Out(), "unable to parse %q, not a valid Template but %s\n", sourceName, reflect.TypeOf(infos[i].Object))
-			continue
+			return fmt.Errorf("unable to parse %q, not a valid Template but %s\n", sourceName, reflect.TypeOf(infos[i].Object))
```

Comment:
You seem to making this the only command that we have where a failure to parse a single item will cause the short-circuiting of every other item.  Contrast with `replace` or `create`.

Why is this command special?